Instagram/View Controllers/DetailsViewController.h

```diff
@@ -7,7 +7,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 @property (nonatomic, strong) Post *post;
 
-- (void)refreshData:(Post *)post;
+- (void)setData:(Post *)post;
```

Comment:
One nit on this is that `Data`/`NSData` is a specific type in ObjC so you could reasonably read this method and expect that to be the type that is being set. I would probably call this `setPost` since it fits the naming conventions better and is more descriptive.